README.md

```diff
@@ -21,7 +21,7 @@ Organizations should first choose the boilerplate that is the best fit to their
 The core requirements for disclose.io's definition of **Full Safe Harbor** are for the policy to provide:
 - Authorization against anti-hacking laws
 - Exemption from anti-circumvention laws 
-- Exemption from violation of the TOS/AUP during security testing
+- Exemption from violation of the Terms of Service (TOS) or Acceptable Use Policy (AUP) during security testing
 - A statement acknowledging good-faith.
 
 The intention of Safe Harbor language provided by disclose.io is for it to be followed specifically, with minor, if any, modifications. If modifications are made, the four tenets laid out above are the most important to address in your policy.
```

Comment:
Important reminder: Authorizations and exemptions from legal liability only cover what is in scope and under the control of the organization using the policy. For example, an organization using disclose.io terms may authorize you to research their own systems, but they cannot authorize you to research systems they do not own. Likewise, an organization using disclose.io terms cannot prevent a third party lawsuit if you breach the third party’s systems or data. Researchers and organizations are still expected to comply with all applicable laws.

Reply:
Hey @its-a-lisa - We integrated a bunch of your feedback into the new PR manually, since so much of the other feedback came in through comments (as opposed to PRs). I'll close this PR now, but please feel free and welcome to put other PRs in if you feel there's anything that was missed, or if you have additional input. Thank you!!!